core/domain/exp_jobs_one_off_test.py

```diff
@@ -1127,23 +1130,28 @@ def test_migration_job_audit_success(self):
             'param_changes': [],
             'solicit_answer_details': False,
             'classifier_model_id': None
-        }).to_dict()
+        }
 
         self.create_exploration_with_states_schema_version(
-            feconf.CURRENT_STATE_SCHEMA_VERSION - 1,
+            36,
             self.NEW_EXP_ID,
             self.albert_id,
             {'Introduction': states_dict}
         )
 
-        job_id = exp_jobs_one_off.ExplorationMigrationAuditJob.create_new()
-        exp_jobs_one_off.ExplorationMigrationAuditJob.enqueue(job_id)
-        self.process_and_flush_pending_tasks()
+        swap_states_schema_ver = self.swap(
+            feconf, 'CURRENT_STATE_SCHEMA_VERSION', 37)
+        swap_exp_schema_ver = self.swap(
```

Comment:
ver --> version

(Don't abbreviate. Here and elsewhere.)

Reply:
Done